app/webpacker/components/RegistrationsV2/RegistrationEdit/RegistrationEditor.jsx

```diff
@@ -199,13 +201,40 @@ export default function RegistrationEditor({ competitor, competitionInfo }) {
           </Message>
         )}
         {registrationEditDeadlinePassed && (
+          // i18n-tasks-use t('registrations.errors.edit_deadline_passed')
```

Comment:
you only need i18n-tasks-use for dynamic uses. This should be fine

Reply:
Changed, thanks!